packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -140,6 +141,20 @@ export const GenericCard = forwardRef(function GenericCard(
       <div className={'SRC-portalCardMain'}>
         {icon}
         <div className="SRC-cardContent">
+          {cardTopButtons && (
+            <Box
+              sx={{
+                position: 'absolute',
+                right: '24px',
+                display: 'flex',
+                float: 'right',
+                flexDirection: 'row',
+                gap: '10px',
+              }}
+            >
+              {cardTopButtons}
+            </Box>
+          )}
```

Comment:
Move buttons to top of the card and `float: right` - this does not seem to cause any layout/overlap issues at narrow screen sizes.